lib/rules/prefer-object-spread.js

```diff
@@ -196,6 +196,34 @@ function autofixObjectLiteral(node, sourceCode) {
     };
 }
 
+/**
+ * Check if the node has modified a given variable
+ * @param {ASTNode|null} node - The node that the rule warns on, i.e. the Object.assign call
+ * @returns {boolean} - true if node is an assignment, variable declaration, or import statement
+ */
+function isModifier(node) {
+    if (!node.type) {
+        return false;
+    }
+
+    return node.type === "AssignmentExpression" ||
+    node.type === "VariableDeclarator" ||
```

Comment:
Would you mind indenting this line and the next line? (Not 100% sure why the indent rule isn't flagging this)

Reply:
oh weird, good catch